### ISSUE TITLE
KeyError: 'AccountLock'

### ISSUE THREAD

**Issue by LRMAN0989** [FriendsOfGalaxy/galaxy-integration-battlenet#31]
**Describe the bug**
If your account is locked in a game (or at least for World of Warcraft; my account was hijacked a few years ago and got my WoW account locked, never found the motivation to get it unlocked because I don't play), it will return a license of "AccountLock" which stops importing

I'm new to github so not sure how to submit changes to the code, but all I had to do to fix it was add "AccountLock" to the license map in definitions.py.

**Expected behavior**
It should either skip over the locked account, or include it regardless of locked status.

**Attached logs**
[plugin-battlenet-ba170431-0649-482f-863b-d248592f1842.log](https://github.com/FriendsOfGalaxy/galaxy-integration-battlenet/files/3948749/plugin-battlenet-ba170431-0649-482f-863b-d248592f1842.log)

**Comment by FriendsOfGalaxy** [comment]
Ok, thanks for report, I'll add "AccountLock" to License_Map

**Comment by FriendsOfGalaxy** [comment]
@LRMAN0989 have you tested this?

```
License_Map = {
    None: LicenseType.Unknown,
    "Trial": LicenseType.SinglePurchase,
    "Good": LicenseType.SinglePurchase,
    "Inactive": LicenseType.SinglePurchase,
    "Banned": LicenseType.SinglePurchase,
    "Free": LicenseType.FreeToPlay,
    "AccountLock": LicenseType.SinglePurchase
}
```

**Comment by LRMAN0989** [comment]
@FriendsOfGalaxy Yep that's exactly what I changed and it imported everything properly afterwards

**Comment by FriendsOfGalaxy** [comment]
good, patch on the way. Will be released in 0.44 when more changes appears
